examples/vue-app/src/components/HelloWorld.vue

```diff
@@ -356,216 +390,347 @@ const topup = async () => {
 };
 
 const debugConsole = async (text: string) => {
-  document.querySelector("#console > p")!.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
+  // document.querySelector("#console > p")!.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
+  consoleDiv.value.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
```

Comment:
if(consoleDiv.value)
    consoleDiv.value.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
    
    use this, ts error is being throwed in local IDE

Reply:
resolved